src/audio/host-zephyr.c

```diff
@@ -442,7 +442,6 @@ static inline bool stream_sync(struct host_data *hd, struct comp_dev *dev)
  */
 static int host_copy_normal(struct host_data *hd, struct comp_dev *dev, copy_callback_t cb)
 {
-	struct comp_buffer __sparse_cache *buffer_c;
```

Comment:
Ok, this is good as host_copy_normal() should always run on the same core as the buffer copier (and the copier would be the point where we would copy to another core). @marcinszkudlinski @lyakh fyi.